controllers/gateway/transform/transformer.go

```diff
@@ -98,46 +103,96 @@ func (toc *tOperationCtx) transform(r *http.Request) (*sv1alpha.Event, error) {
 		},
 		Payload: tp.Payload,
 	}
+
+	toc.log.Debug().Interface("cloud-event", ce).Msg("transformed event")
 	return ce, nil
 }
 
-// dispatches the event to configured sensor
+// getWatcherIP returns IP of service which backs given component.
+func (toc *tOperationCtx) getWatcherIP(name string) (string, string, error) {
```

Comment:
this method seems a bit odd to me.. shouldn't we be using DNS with service names instead of inspected for IP addresses?